deeds-dapp-webapp/src/main/webapp/vue-app/layout/components/Topbar.vue

```diff
@@ -17,7 +17,12 @@
  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 -->
 <template>
-  <v-toolbar elevation="0">
+  <v-app-bar
+    :fixed="isMobile"
+    color="white"
+    :elevation="!isMobile && '0'"
+    elevate-on-scroll
+    :height="isMobile && '64px'">
```

Comment:
```suggestion
    :height="isMobile && '64px'"
    :elevate-on-scroll="isMobile">
```

Two remarks : 
1. Add variabilized properties before the fixed values one ( `:xxx="yyy"` before the properties `xxx="yyy"`, without `:` )
2. The elevation on scroll should be applied on Mobile only knowing that it will be fixed on Mobile view only, isn't it ?

Reply:
fixed